### ISSUE TITLE
Android Sample project crashes at start up

### ISSUE THREAD

**Issue by jcarbo** [cocos2d/cocos2d-x#19290]
- cocos2d-x version: 3.17.1
- devices test on: Samsung Galaxy S6, Samsung Galaxy S8+
- developing environments
   - NDK version: 18.1.5063045
   - Android Studios version: 3.2.1
   - Gradle version: 4.6
   - Gradle plugin version: 3.1.0
   - OS: masOS High Sierra Version 10.13.6

Steps to Reproduce:

1. Change PROP_BUILD_TYPE to ndk-build
2. Gradle Sync, Clean, Rebuild and Run CppTests

For the record, if the PROP_BUILD_TYPE is cmake then the Gradle sync fails.

Stacktrace:
`
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG: backtrace:
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #00 pc 00d77bf4  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN8TestList16tableCellAtIndexEPN7cocos2d9extension9TableViewEi+223)
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #01 pc 018d366d  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN7cocos2d9extension9TableView17updateCellAtIndexEi+124)
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #02 pc 018d51a3  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN7cocos2d9extension9TableView19scrollViewDidScrollEPNS0_10ScrollViewE+1174)
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #03 pc 018cd54f  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN7cocos2d9extension10ScrollView16setContentOffsetENS_4Vec2Eb+490)
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #04 pc 018d2baf  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN7cocos2d9extension9TableView18_updateContentSizeEv+294)
2019-01-09 09:18:17.434 16013-16013/? A/DEBUG:     #05 pc 00d774b3  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN19TestCustomTableView6createEPN7cocos2d9extension19TableViewDataSourceENS0_4SizeE+154)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #06 pc 00d76d2b  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN8TestList11runThisTestEv+94)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #07 pc 012b92b3  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN14TestControllerC1Ev+254)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #08 pc 012bffcf  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN14TestController11getInstanceEv+70)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #09 pc 00d75b3b  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN11AppDelegate29applicationDidFinishLaunchingEv+3338)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #10 pc 01540c99  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (_ZN7cocos2d11Application3runEv+18)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #11 pc 0135a203  /data/app/org.cocos2dx.cpp_tests-2/lib/arm/libcpp_tests.so (Java_org_cocos2dx_lib_Cocos2dxRenderer_nativeInit+210)
2019-01-09 09:18:17.435 16013-16013/? A/DEBUG:     #12 pc 0013640d  /data/app/org.cocos2dx.cpp_tests-2/oat/arm/base.odex (offset 0x12a000)
2019-01-09 09:18:18.070 3138-3138/? E/lowmemorykiller: Error writing /proc/15971/oom_score_adj; errno=22
`

**Comment by Dimon4eg** [comment]
> cocos2d-x version: 3.7.1

3.17.1 maybe?

**Comment by jcarbo** [comment]
@Dimon4eg Yes, version 3.17.1. I updated the ticket. Sorry for my mistake.
